### PR TITLE
dashbard & wallet to wallet

### PR DESCRIPTION
**Pull request by BirtukanDegu**
wallet to wallet money transfer page and dashboard has been added.

### CODE REVIEW

**Review by moheranus — DISMISSED** [review]
nice one

File: tolopay/src/Components/Dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        </div>
+    </Section>
+  )
+}
+
+const Section = styled.section`
+    margin-left: 10vw;
+    padding: 2rem;
+    padding-left: 0rem;
+    height: 100%
```
Comment (BisratGirma) [inline tolopay/src/Components/Dashboard/Dashboard.jsx]:
```suggestion
    height: 100%;
```

File: tolopay/src/Components/Dashboard/Chart.jsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
+import React from "react";
+import styled from "styled-components";
+import { AreaChart, Area, Tooltip, ResponsiveContainer } from "recharts";
```
Comment (BisratGirma) [inline tolopay/src/Components/Dashboard/Chart.jsx]: include your package.json file in your commit

**Review by BisratGirma — CHANGES_REQUESTED** [review]
please include your package.json file in your commit 

you have used some packages like 'rechart' that are not on main package.json file

**Review by Yonasketema — DISMISSED** [review]
great work !

**Review by BisratGirma — DISMISSED** [review]

**Review by BisratGirma — APPROVED** [review]

**Pull request merged**